### PR TITLE
Add ZeroTier One 1.0.3

### PR DESCRIPTION
**Pull request by SeanSith**
Fulfilling #11249, a request for a cask for ZeroTier One.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Could you explain the commented code?

**Comment by SeanSith** [comment]
Yes, I can: lazy committing after I yanked code from 'gpgtools' and neglected to commit the removal of those comments. Very sorry. I've removed it and pushed.

File: Casks/zerotier-one.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask :v1 => 'zerotier-one' do
+  version '1.0.3'
+  sha256 '2822ecec169e76503e69fb3cd546a0ea91066f29e1832fbe233e5226bc90eb42'
+
+  url 'https://www.zerotier.com/dist/dist/ZeroTier%20One.pkg'
+  name 'ZeroTier One'
+  homepage 'https://www.zerotier.com'
+  license :gpl
+
+  container :type => :pkg
```
Comment (vitorgalvao) [inline Casks/zerotier-one.rb]: Are you sure this is needed?

**Comment by vitorgalvao** [comment]
There’s some trailing whitespace that needs to be removed.

File: Casks/zerotier-one.rb (hunk 1 above)
Comment (SeanSith) [inline Casks/zerotier-one.rb]: Apparently not. Guess I need to understand the 'container' directive better. Line removed and cask tested.

**Comment by SeanSith** [comment]
Added newline at the end of the file and remove spurious whitespace.

**Comment by vitorgalvao** [comment]
You added an empty line, instead of the required newline. It was fine as it was.

File: Casks/zerotier-one.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  url 'https://www.zerotier.com/dist/dist/ZeroTier%20One.pkg'
+  name 'ZeroTier One'
+  homepage 'https://www.zerotier.com'
+  license :gpl
+
+  pkg 'ZeroTier One.pkg'
+
+  uninstall :pkgutil => 'com.zerotier.*'
+
```
Comment (vitorgalvao) [inline Casks/zerotier-one.rb]: This one should also be removed.

**Comment by SeanSith** [comment]
Sorry. I promise I'm taking notes.

**Comment by vitorgalvao** [comment]
Nothing to apologise for. Your turnaround time for fixes is great. Just notices something else, though, apologies.

Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

**Comment by SeanSith** [comment]
Just want to be useful. I really love the work everyone's done with caskroom and want to give back. 

Adjustments made and pushed.

**Comment by vitorgalvao** [comment]
Looking great. Thank you.

**Pull request merged**